### PR TITLE
Delete event

### LINKED ISSUE #54: US: Delete/Destroy existing Event
Mutations

Delete/Destroy an existing Event

### PR DESCRIPTION
**Pull request by LHJE**
### Code Highlights:
 * Adds test to delete Events
 * Adds ability to delete Events
### Where should the reviewer start?
 * At the `delete_event_spec.rb`
### Have Tests Been Added?
- [ ] No.
- [ ] Yes, but all tests are not passing.
- [x] Yes, and all are passing.
### Issues:
* Addresses #54 
* Closes #54 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion
- [x] Rubocop Violations: the same `get_user` error

### CODE REVIEW

File: spec/requests/graphql/mutations/delete_event_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      event_id = user.events[0].id
+
+      post graphql_path, params: { query: query(event_id) }
+      result = JSON.parse(response.body)
+
+      data = result['data']['deleteEvent']
+
+      expect(data['id']).to eq("#{event_id}")
+      expect(Event.all.count).to eq(2)
+      expect(Event.where(id: event_id)).to eq([])
```
Comment (leahriffell) [inline spec/requests/graphql/mutations/delete_event_spec.rb:16]: good tests

**Review by leahriffell — APPROVED** [review]

**Review by AngelaGuardia — APPROVED** [review]

File: spec/requests/graphql/mutations/update_event_spec.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Mutations::Events::UpdateEvent, type: :request do
+  describe '.resolve' do
+    it 'updates a user' do
```
Comment (leahriffell) [inline spec/requests/graphql/mutations/update_event_spec.rb:5]:
```suggestion
    it 'updates an event' do
```

File: spec/requests/graphql/mutations/update_event_spec.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      user = create(:user, :with_eras_events)
+      event_id = user.events[0].id
+
+      post graphql_path, params: { query: query(event_id) }
+      result = JSON.parse(response.body)
+
+      data = result['data']['updateEvent']
+
+      expect(data['name']).to eq('Paleo gluten-free aesthetic everyday farm-to-table slow-carb distillery.')
```
Comment (dhalverson) [inline spec/requests/graphql/mutations/update_event_spec.rb:14]: Will these work if we re-seed? Won't the fields be random?

**Review by dhalverson — APPROVED** [review]
Reviewed and approved

**Pull request merged**